.github/workflows/add-qa-link-to-issues.yml

```diff
@@ -1,17 +1,20 @@
 name: Add QA link to issues on successful build
 
 on:
-  deployment_status:
-    state: success
+  pull_request:
 
 jobs:
   add-comment:
     runs-on: ubuntu-latest
     permissions:
-      issues: write
+      pull-requests: write
     steps:
-      - name: Add comment
-        uses: peter-evans/create-or-update-comment@a35cf36e5301d70b76f316e867e7788a55a31dae
+      - name: Extract branch name
+        shell: bash
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_HEAD_REF#refs/heads/})"
```

Comment:
I've recently seen warnings that set-output will be deprecated soon. https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/.